packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -275,6 +218,9 @@ function MongoNav({
       orgSettings: `${hosts.cloud}/v2#/org/${data?.currentOrganization?.orgId}/settings/general`,
     },
     orgNav: {
+      leaf: data?.currentOrganization
+        ? `${hosts.cloud}/v2#/org/${data?.currentOrganization?.orgId}/`
```

Comment:
```suggestion
        ? `${hosts.cloud}/v2#/org/${data.currentOrganization?.orgId}/`
```

Can drop that one as you checked it in the ternary. It'd be nice to not also have to check if `currentOrganization` exists, and do that in the first part of the ternary too.

Reply:
Isn't this checking that currentOrg exists in the first part of the ternary?